pkg/client/client.go

```diff
@@ -86,6 +86,8 @@ type Service interface {
 	HostService(host string) HostOrchestratorService
 
 	RootURI() string
+
+	AuthHeader() (http.Header, error)
```

Comment:
The code that establishes the websocket connection should be part of the `HostOrchestratorService`, so that it can be used from places other than the cli.

With that change there would be no need to expose the `AuthHeader` in the CO client, it can simply pass it to the HO client behind the scenes.

I understand this is a big change, so it's ok to do it in another PR.

Reply:
Implemented WebSocket connection at the HO client and changed to use it when possible.

Thanks!